tests/cases/fourslash/switchCaseCompletionPriority.ts

```diff
@@ -0,0 +1,125 @@
+/// <reference path="fourslash.ts" />
+
+//// declare const thing: string;
+
+//// // Basic switch block - should prioritize case/default
+//// switch (thing) {
+////     /*basic*/
+//// }
+
+//// // Same - show all completions (not at switch body level)
+//// switch (thing) {
+////   case 42:
+////   /*sameAll1*/
+//// }
+
+//// // Same - show all completions (after break at same column as case)
+//// switch (thing) {
+////   case 42:
+////   break;
+////   /*sameAll2*/
+//// }
+
+//// // Same - show all completions (complex nested structure)
+//// switch (thing) {
+////   case 42:
+////   if (Math.random()) {
+////   }
+////   else {
+////   }
+////   /*sameAll3*/
+//// }
+
+//// // NEW - prioritize case/default (after break on different column)
+//// switch (thing) {
+////   case 42:
+////     break;
+////   /*newPrio1*/
+//// }
+
+//// // NEW - prioritize case/default (break on same line)
+//// switch (thing) {
+////   case 42: break;
+////   /*newPrio2*/
+//// }
+
+//// // NEW - prioritize case/default (after return)
+//// switch (thing) {
+////   case 42:
+////     return 1;
+////   /*newPrio3*/
+//// }
+
+//// // NEW - prioritize case/default (after throw)
```

Comment:
Remove these NEW labels, don't claim what to expect. Instead, describe the positioning of these.